migrations/2_deploy_contracts.js

```diff
@@ -159,7 +181,7 @@ module.exports = function (deployer, network, accounts) {
     const poolTx = await deployer.deploy(...deploymentArgs, { gasPrice: gasPrice });
     const ggInstance = await goodGhostingContract.deployed();
 
-    if (config.deployConfigs.owner && config.deployConfigs.owner != "0x") {
+    if (config.deployConfigs.owner && config.deployConfigs.owner != "0x" && config.deployConfigs.owner != "") {
```

Comment:
the previous condition already considered empty string (this part, `config.deployConfigs.owner`, already covers empty string and any other falsy values like `undefined`, `null`, `0`).
```suggestion
    if (config.deployConfigs.owner && config.deployConfigs.owner != "0x") {
```